node-js-buy/views/index.pug

```diff
@@ -14,23 +14,24 @@ html(lang='en', data-framework='javascript')
           h2=shop.description
 
       .Product-wrapper
-        each product in products
-          .Product
-            img(src=product.images[0].src)
-            h5.Product__title=product.title
-            span.Product__price=`$ ${product.variants[0].price}`
-            form(method='post' action='/add_line_item/' + product.id)
-              each option in product.options
-                label=option.name
-                select.Product__option(name=option.name)
-                  each value in option.values
-                    option(value=value.valueOf())
-                      =value
-              label Quantity
-              input.Product__quantity(min='1', type='number', name='quantity', value='1')
-              input(type='hidden' name='checkoutId' value=cart.id)
-              button.Product__buy.button#add-to-cart-button(type='submit')
-                  | Add to cart
+        each collection in collections
```

Comment:
Is this change needed?

I get an error:

```plain
TypeError: /Users/scott/dev/storefront-api-examples/node-js-buy/views/index.pug:17
    15| 
    16|       .Product-wrapper
  > 17|         each collection in collections
    18|           each product in collection.products
    19|             .Product
    20|               img(src=product.images[0].src)

Cannot read property 'length' of undefined
```

Reply:
Thank you for pointing this out @swalkinshaw.  This is a mistake on my part.  I am using this as the basis for my own app and missed backing this change out.  I'll fix it and push in the next few minutes.